app/graphql/resolvers/customer/customer.js

```diff
@@ -0,0 +1,24 @@
+import {
+  transformPersonRolesToCustomerAuthorisedBusinessesRoles,
+  transformPersonPrivilegesToCustomerAuthorisedBusinessesPrivileges,
+  transformPersonSummaryToCustomerAuthorisedBusinesses
+} from '../../../transformers/rural-payments-portal/customer.js'
+
+export const Customer = {
+  async authorisedBusinesses ({ id }, __, { dataSources }) {
+    const summary = await dataSources.ruralPaymentsPortalApi.getPersonSummaryByPersonId(id)
+    return transformPersonSummaryToCustomerAuthorisedBusinesses(id, summary)
+  }
+}
+
+export const CustomerAuthorisedBusiness = {
+  async roles ({ id, customerId }, __, { dataSources }) {
+    const authorisation = await dataSources.ruralPaymentsPortalApi.getAuthorisationByOrganisationId(id)
+    return transformPersonRolesToCustomerAuthorisedBusinessesRoles(customerId, authorisation.personRoles)
+  },
+
+  async privileges ({ id, customerId }, __, { dataSources }) {
+    const authorisation = await dataSources.ruralPaymentsPortalApi.getAuthorisationByOrganisationId(id)
+    return transformPersonPrivilegesToCustomerAuthorisedBusinessesPrivileges(customerId, authorisation.personPrivileges)
+  }
+}
```

Comment:
We agreed were going to start writing some functional tests to cover the resolver, do you mind adding these?